shivyc/tree/nodes.py

```diff
@@ -600,6 +600,8 @@ def _generate_array_ctype(self, decl, prev_ctype):
 
     def _generate_func_ctype(self, decl, prev_ctype):
         """Generate a function ctype from a given a decl_node."""
+        # save identifiers
+        identifiers = []
```

Comment:
Could you do this more easily on line 546 instead?

Reply:
Are you mean checking redefinition of identifier inside make_ctype?

Reply:
Fix this on #104